spec/services/geo_discovery/document_builder_spec.rb

```diff
@@ -396,4 +398,19 @@
       end
     end
   end
+
+  describe "raster set" do
+    let(:geo_work) do
+      FactoryBot.create_for_repository(
+        :raster_set,
+        coverage: coverage.to_s
+      )
+    end
+
+    it "has a wmts reference" do
+      geo_work
+      refs = JSON.parse(document["dct_references_s"])
+      expect(refs["http://www.opengis.net/def/serviceType/ogc/wmts"]).to eq "https://map-tiles-test.example.com/mosaicjson?id=#{geo_work.id}"
```

Comment:
Does this need to be `https://map-tiles-test.example.com/mosaicjson/WMTSCapabilities.xml?id=#{geo_work.id}`

Reply:
answer: yes. this has been updated.